CHANGELOG.md

```diff
@@ -12,6 +13,7 @@
 
 ### 💡 Enhancements 💡
 
+- Enable Persistent Queue by default (#5711)
```

Comment:
This is a bit misleading. We don't enable persistent queue by default. We just make the feature available in the default build.